.github/workflows/docker-image.yml

```diff
@@ -13,25 +13,34 @@ on:
   release:
     types: [published]
 
+env:
```

Comment:
We can change this to environment variables defined in GitHub entirely? This way I can also publish images for my own use.

Reply:
I am all for this @DanielHabenicht. However, is there a way we can define multiple registries? I'd like to be able to publish this to ghcr.io as well as docker hub.

Reply:
yeah, I think I can add a condition for the publish to docker so its only executed on this repository